hrrecruitment/CRM/HRRecruitment/Upgrader.php

```diff
@@ -16,4 +16,21 @@ public function upgrade_1400() {
     return TRUE;
   }
 
+  /**
+   * Renames the main menu item "Vacancies" to "Recruitment"
+   *
+   * @return bool
+   */
+  public function upgrade_1401() {
```

Comment:
The hrrecruitment upgrader doesn't run the upgraders during the installation, so you'll need to also update the item label [here](https://github.com/civicrm/civihr/blob/staging/hrrecruitment/hrrecruitment.php#L102-L103), otherwise new installation will still display the old label